src/Illuminate/Foundation/Bootstrap/DetectEnvironment.php

```diff
@@ -16,6 +16,11 @@ public function bootstrap(Application $app)
 	{
 		try
 		{
+			if( "mutable" == getenv('VAR_MUT') )
```

Comment:
Looking at the [L5 contribution guidelines](http://laravel.com/docs/5.0/contributions#coding-style) I'm not sure what I need to change. Should I be using `===`?